### DISCUSSION TITLE
Custom Meta and Placeholders

### DISCUSSION THREAD

**Discussion by LilSadPanda** [Zrips/CMI#8115] [Feature Requests]
The ability to use placeholders as color codes.

For example you can use Deluxemenus meta placeholder to store a hex value and then use the placeholder 

For example
`%deluxemenus_meta_hexcolor1_STRING_%` = #00ffd4
`%deluxemenus_meta_hexcolor2_STRING_%` = #00ff44

And then use it in a hologram or message like this:
`{%deluxemenus_meta_hexcolor1_STRING_%>}`test`{%deluxemenus_meta_hexcolor2_STRING_%<}`

Maybe Zrips can make his own meta placeholder so we can store custom data with CMI

**Comment by dkalaxdk** [comment]
You can already achieve that using the usermeta placeholders.

You can see an example of how it can be achieved [here](https://github.com/mrfdev/CMI/blob/master/Resources/Add-ons/chatcolortags.md).
